prow/presubmit.yaml

```diff
@@ -76,8 +76,10 @@ kubernetes/kubernetes:
 
 - name: pull-kubernetes-cross
   context: Jenkins Cross Build
-  rerun_command: "@k8s-bot build this"
-  trigger: "@k8s-bot (cross )?build this"
+  always_run: true
+  skip_report: true
+  rerun_command: "@k8s-bot cross test this"
```

Comment:
We have had `@k8s-bot build this`, can we set rerun_command to `@k8s-bot cross-build this` 
and the trigger to `@k8s-bot (cross[ -]?)?build this` 
(not sure I got the syntax right, but it should allow `crossbuild`, `cross-build`, `cross build` and `build`)

Sounds good?